functional_tests/steps/methodology_page.py

```diff
@@ -0,0 +1,129 @@
+from behave import given, then, when  # pylint: disable=no-name-in-module
+from behave.runner import Context
+from playwright.sync_api import expect
+
+
+@given("the user creates a methodology page as a child of the existing topic page")
+@when("the user creates a methodology page as a child of the existing topic page")
```

Comment:
```suggestion
@step("the user creates a methodology page as a child of the existing topic page")
```
Lets you match the step wording on any prefix

Reply:
Resolving so I'm not blocking while away

Reply:
[9755549](https://github.com/ONSdigital/dis-wagtail/pull/102/commits/9755549f29603593566656eadbce80d9a3e04ae9)